Gruntfile.js

```diff
@@ -92,6 +92,7 @@ module.exports = function(grunt) {
   // on each recompile
   var updateTsFiles = function() {
     tsFiles = grunt.file.read("src/reference.ts")
+      .replace(/\r\n/g, "\n")
```

Comment:
This is likely a precision error due to being on a different operating system (This looks like a text measurement error and text is known to be very inconsistently measured across browsers).  If you run into any more errors moving forward.  Feel free to skip them in development if it causes trouble so we can inspect the test to hopefully make it less fragile.